### DISCUSSION TITLE
Update FIT SDK to version 21.54

### DISCUSSION THREAD

**Discussion by lifof** [cagnulein/qdomyos-zwift#346] [Ideas]
@cagnulein 
The version included now is the 21.40, I don't know there could be new functionality or bug fixes in the newest version, should we start by creating a new branch for this ? or is it optional ? I need to know so that I could hunt some warnings In both cases
( btw in the codeRefactoring branch I didn't touch the files in fit-sdk folder ( they were only formatted ) even though the compiler complains about them, my goal is to have code that produces zero warnings )

**Comment by cagnulein** [comment]
If you want to try to see if it produces less warning go on, but i guess they will stay there :D
Maybe we can put these files in a library in order to don't compile them every time.

**Reply by lifof** [reply]
Putting them in a lib format will be troublesome cause we'll need to create one for each and every platform, OS and architecture.
I think that we should put them in a repository then add them as a git sub-module, that way we can maintain them separately.
I think it'll be nice that way.

**Reply by cagnulein** [reply]
but in this way you will have always the warning stuff when you will build it :)
i was saying about the library just in order to avoid warnings :D
or maybe we can fix the warnings in the sdk too :)

**Reply by lifof** [reply]
Yeah when we'll start maintaining it the warnings will disappear :wink:
